Nasal/engine.nas

```diff
@@ -293,6 +274,15 @@ var carb_icing_function = maketimer(1.0, func {
     };
 });
 
+# =========== engine damage =======================
+var damage_vol_eff_factor = 0.0;
+var damage = getprop("/engines/active-engine/damage-level");
+
+var engine_damage_function = maketimer(1.0, func {
```

Comment:
Why not convert this function to JSBSim as well? There's also a tab on the next line (282)

Reply:
Right.